pandas/tests/frame/test_to_csv.py

```diff
@@ -413,12 +444,15 @@ def test_to_csv_from_csv_w_all_infs(self):
 
         with ensure_clean() as path:
             self.frame.to_csv(path)
-            recons = DataFrame.from_csv(path)
 
-            # TODO to_csv drops column name
-            assert_frame_equal(self.frame, recons, check_names=False)
-            assert_frame_equal(np.isinf(self.frame),
-                               np.isinf(recons), check_names=False)
+            with tm.assert_produces_warning(FutureWarning,
+                                            check_stacklevel=False):
```

Comment:
i would fix almost all of these to just call read_csv (maybe leave a couple which catch the earning)

Reply:
Yeah, that makes sense.

Reply:
also any test that is not for to_csv could be moved to the parsers testing
prob are duplicates (of existing tests)

can be done in a follow up if u want (make an issue in that case)

Reply:
These are all for `to_csv`.  They are round-trip checks mostly.